contributed_definitions/NXstage_lab.nxdl.xml

```diff
@@ -154,6 +154,7 @@
     </field>
     <!--NEW ISSUE: rather the field if possible should be specified-->
     <group type="NXpositioner"/>
+    <group type="NXsensor"/>
```

Comment:
Let us harmonize with NXmpes, where NXactuator has also been used.

Reply:
Agreed but in a separate PR, here added NXactuator to start this process as indeed when one has a sensor one could also have an actuator. There is a comment from @lukaspie about stage_lab and multi-stage in MPES, I am okay with consolidating but in a separate PR, everybody could have already commented on that NXstage_lab which has essentially been sitting here also with the contextualizing literature for now almost 1,5 years, good that you give feedback showing that finally somebody cares :)